src/data/_contributors.yaml

```diff
@@ -547,6 +547,38 @@ ilyagrigorik:
   description:
     en: Ilya is a Developer Advocate and Web Perf Guru
 
+jgruber:
```

Comment:
Let’s sort these entries alphabetically, per the instructions at the top of the file — otherwise the bots won’t let me merge it.